### ISSUE TITLE
Brawlout (263720)

### ISSUE THREAD

**Issue by mimattr** [ValveSoftware/Proton#2154]
# Compatibility Report
- Name of the game with compatibility issues: **Brawlout**
- Steam AppID of the game: **263720**

## System Information
- GPU: **GTX 970**
- Driver/LLVM version: **NVIDIA 415.25**
- Kernel version: **4.4.0-141-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **3.16-6 Beta**

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-263720.log](https://gist.github.com/mimattr/6858a74e2807f2a02421ddef5114afcb)

## Symptoms
The application appears to work as expected except for 2 issues:
1. Steam Controller is not detected/working (official wired Xbox 360 Controller functions correctly)
1. Exiting results in an application crash and defunct process ([error.log](https://gist.github.com/mimattr/8b843566507107fadc5402c175970ced))

## Reproduction
1. Select the title within your `Library` using the Steam client and press the `Play` button

**Comment by eeyrjmr** [comment]
I can confirm this. Recently purchased this to try and hit this issue.

https://steamcommunity.com/app/263720/discussions/0/154644045357618420/?ctp=2#c1744480966992666974

**Comment by dragonfly-lover** [comment]
I also Can confirm both.

**Comment by mimattr** [comment]
# Whitelist Request

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 430.40**
- Kernel version: **5.0.0-23-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **4.11-2**

## I confirm:
- [x] that pressing the `Play` button in the Steam client is sufficient.
- [ ] that [runtime config options](https://github.com/ValveSoftware/Proton#runtime-config-options) are necessary to run the game.
- [ ] that no workarounds other than PROTON_* env variables are necessary.

## Issues
- [x] I haven't experienced any issues.
- [x] There are no issues left open for this game.
- [ ] Although I consider the gaming experience equal to Windows there are remaining issues:

**Comment by kisak-valve** [comment]
Hello @eeyrjmr, @dragonfly-lover, are either of you seeing an issue with this game and Proton 4.11 that would block it from being whitelisted?

**Comment by dragonfly-lover** [comment]
nope. i tested it again just now with the latest proton and everything runs fine.
